doc/api/async_hooks.md

```diff
@@ -282,10 +282,10 @@ TCPWRAP(4): trigger: 2 execution: 0
 The `TCPSERVERWRAP` is the server which receives the connections.
 
 The `TCPWRAP` is the new connection from the client. When a new
-connection is made the `TCPWrap` instance is immediately constructed. This
-happens outside of any JavaScript stack (side note: a `executionAsyncId()` of
-`0` means it's being executed from C++, with no JavaScript stack above it).
-With only that information, it would be impossible to link resources together in
+connection is made, the `TCPWrap` instance is immediately constructed. This
+happens outside of any JavaScript stack. (An `executionAsyncId()` of `0` means
```

Comment:
The full sentence in parens followed by a period outside of the parens looks a little bit odd.